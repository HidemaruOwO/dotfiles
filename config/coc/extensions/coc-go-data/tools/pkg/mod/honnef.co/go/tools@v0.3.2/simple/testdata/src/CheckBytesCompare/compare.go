package pkg

import "bytes"

func fn() {
	_ = bytes.Compare(nil, nil) == 0 //@ diag(` bytes.Equal`)
	_ = bytes.Compare(nil, nil) != 0 //@ diag(`!bytes.Equal`)
	_ = bytes.Compare(nil, nil) > 0
	_ = bytes.Compare(nil, nil) < 0
}
