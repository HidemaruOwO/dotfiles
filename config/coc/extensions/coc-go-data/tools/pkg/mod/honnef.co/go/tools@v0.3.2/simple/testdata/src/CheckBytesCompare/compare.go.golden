package pkg

import "bytes"

func fn() {
	_ = bytes.Equal(nil, nil)  //@ diag(` bytes.Equal`)
	_ = !bytes.Equal(nil, nil) //@ diag(`!bytes.Equal`)
	_ = bytes.Compare(nil, nil) > 0
	_ = bytes.Compare(nil, nil) < 0
}
